tensorflow_addons/custom_ops/opticalflow/BUILD

```diff
@@ -0,0 +1,26 @@
+licenses(["notice"])  # Apache 2.0
+
+package(default_visibility = ["//visibility:public"])
+
+cc_binary(
+    name = "_correlation_cost_ops.so",
+    srcs = [
+        "cc/kernels/correlation_cost_op.cc",
+        "cc/kernels/correlation_cost_op.h",
+        "cc/kernels/correlation_cost_op_gpu.cu.cc",
+        "cc/ops/correlation_cost_op.cc",
+    ],
+    copts = [
+        "-pthread",
+        "-std=c++11",
+        "-D_GLIBCXX_USE_CXX11_ABI=0",
```

Comment:
Should this really be hardcoded here?

Reply:
We have a few open tickets about how we're going to refactor this config (e.g., https://github.com/tensorflow/addons/pull/211) to a more central location.  However, it is still unresolved.

Reply:
The ABI? No it shouldn't but we'll be discussing how we want bazel to set this. See #211

Reply:
I don't like it. But it is a copy of the other BUILD files from this repo. My default would be "1" anyway